src/client/components/pages/about.js

```diff
@@ -15,11 +15,16 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+/* eslint valid-jsdoc: "error" */
```

Comment:
What I meant here was it would be better to add the `requiresReturn: false` option to the `.eslintrc` file in the root directory of the repository, to save us from having to add a comment at the top of each file.